srcpkgs/loudgain/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'loudgain'
+pkgname=loudgain
+version=0.6.8
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libebur128-devel ffmpeg-devel taglib-devel"
+short_desc="Versatile ReplayGain 2.0 loudness normalizer,EBU R128/ITU BS.1770"
+maintainer="André Cerqueira <acerqueira021@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/Moonbase59/loudgain"
+distfiles="https://github.com/Moonbase59/loudgain/archive/v${version}.tar.gz"
+checksum=1137c193ad941b366e87c5d84ccc95a7aa8571affc060db0bd1cf72c489aeaee
+
+pre_build() {
+	mkdir -p build && cd build
```

Comment:
This shouldn't be necessary, the cmake build_style already sets up the build directory at configure time before pre_build.
https://github.com/void-linux/void-packages/blob/master/common/build-style/cmake.sh#L6